src/style/font.ts

```diff
@@ -0,0 +1,29 @@
+import { ThemeCSSRule } from '.'
+
+export interface FontTheme {
+  /** Defaults to sans-serif */
+  readonly sans: string
+  /** Defaults to serif */
+  readonly serif: string
+  /** Defaults to monospace */
+  readonly mono: string
```

Comment:
I would just make FontTheme a `Record<string, string>` and let the individual webpage decide.  For example, a webpage may decide to use `{text: 'xxx', header: 'yyy'}` or just `{text: 'xxx'}` or something more complex.  I would not go about customizing it this way.